clippy_lints/src/lib.rs

```diff
@@ -9,7 +9,7 @@
 #![recursion_limit = "512"]
 #![warn(rust_2018_idioms, trivial_casts, trivial_numeric_casts)]
 #![deny(rustc::internal)]
-#![cfg_attr(feature = "deny-warnings", deny(warnings))]
+#![cfg_attr(feature = "deny-warnings", deny(rustc::warnings), deny(clippy::warnings))]
```

Comment:
This turns this line into dead code, since neither `rustc::warnings`, not `clippy::warnings` exist. I blame it on your tiredness, like you said on discord 😉